app/views/images/featured.html.erb

```diff
@@ -1,38 +1,89 @@
+    <div class="search-section">
+      <p>
+        The Marchand Image Archive contains more than 15,000 images with maps, Aztec codices, early American, advertising posters and more.
+      </p>
+      <form name="filter">
+        <div class="input-group keyword-search">
+          <input name="q" class="form-control" placeholder="Search" type="text" value="<%= params[:q] %>">
+          <span class="input-group-addon">
+            <button type="submit">
+              <span class="glyphicon glyphicon-search"></span>
+            </button>
+          </span>
+        </div>
+        <%= render 'advanced_search' %>
+      </form>
     </div>
-    <%= render 'advanced_search' %>
-  </form>
-</div>
 
-<div class="container marchand-description-container">
-  <div class="row marchand-description-content">
-    <div class="col-md-3">
-      <%= image_tag("Marchand.gif", :class => "center-block marchand-profile") %>
+    <!-- Roland Marchand IMAGE -->
+    <div class="box">
+      <div class="imgBx">
+        <img src="https://historyproject.ucdavis.edu/images-and-files/marchand.png" alt="UC Davis White Logo Small LA" class="image-inline" title="Roland Marchand" align="left" />
+      </div>
+      <div class="content">
+        Roland Marchand <br />
+        <i>Historian</i><br />
+        <p><%= link_to about_url do %>
+          <b>Read more about the collection </b><i class="fa fa-angle-double-right"></i>
+        <% end %></p>
+      </div>
     </div>
-    <div class="col-md-9">
-      <div>
-        <h3>Marchand Archive</h3>
-        <p>
-          The Marchand Archive is a collection of more than 15,000 images and a growing number of document-based lesson plans. This site brings together the original <b><i>Adventures in Roland Marchand’s File Cabinent (1999)</i></b> and the <b><i>Marchand Image Collection (2001)</i></b>. In 2007, with funding from the National Endowment for the Humanities, we expanded the image collection with contributions from UC Davis History Department faculty. Many of these images, digitized from slides, are not available elsewhere. The expanding collection of lesson plans developed by teacher leaders, graduate students, and staff, encourage students to apply their analytical skills to a set of primary sources from which they can deduce and explain events from the past.
-        </p>
-        <p>
-          In July 2017, with support from the UC Davis Division of Social Sciences, we launched this updated platform to make using the collection much easier. Please share with us how you are using the collection and contact us if you have any difficulties finding materials you have used in the past.
-        </p>
+  </div>
+
+  <div class="col-md-7">
+    <!-- CAROUSEL -->
+    <div>
+      <div id="myCarousel" class="carousel slide" data-ride="carousel">
+        <!-- Indicators -->
+        <ol class="carousel-indicators">
+          <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
```

Comment:
I'm unable to click on the carousel to go to that collection. Is that working for you? If not, please add it as a feature.